contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -154,26 +124,27 @@ contract LPOptimiserStrategy is DefaultAccessControl {
 
     /// @notice Set new optimimal tick range based on current tick
```

Comment:
small typo: "optimal"

Reply:
based on current twap tick given that we are using the offchain moving average of the fixed rate in the current iteration